build-tools/installers/create-installers.targets

```diff
@@ -0,0 +1,113 @@
+    <MSBuildFiles Include="$(MSBuildSrcDir)\Mono.Android.DebugRuntime-*"/>
+  </ItemGroup>
+  <ItemGroup>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\cross-arm.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\cross-arm64.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\cross-x86.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\cross-x86_64.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\opt.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\llc.exe"></MSBuildFilesWin>
+    <MSBuildFilesWin Include="$(MSBuildSrcDir)\**\proguard.bat"></MSBuildFilesWin>
+  </ItemGroup>
+  <ItemGroup>
+    <MSBuildFilesDarwin Include="$(MSBuildSrcDir)\**\cross-arm"></MSBuildFilesDarwin>
+    <MSBuildFilesDarwin Include="$(MSBuildSrcDir)\**\cross-arm64"></MSBuildFilesDarwin>
+    <MSBuildFilesDarwin Include="$(MSBuildSrcDir)\**\cross-x86"></MSBuildFilesDarwin>
+    <MSBuildFilesDarwin Include="$(MSBuildSrcDir)\**\cross-x86_64"/>
+    <MSBuildFilesDarwin Include="$(MSBuildSrcDir)\**\illinkanalyzer"/>
+    <MSBuildFilesDarwin Include="$(MSBuildSrcDir)\**\mono"></MSBuildFilesDarwin>
+    <MSBuildFilesDarwin Include="$(MSBuildSrcDir)\**\mono-symbolicate"></MSBuildFilesDarwin>
+    <MSBuildFilesDarwin Include="$(MSBuildSrcDir)\**\opt"></MSBuildFilesDarwin>
+    <MSBuildFilesDarwin Include="$(MSBuildSrcDir)\**\llc"></MSBuildFilesDarwin>
+    <MSBuildFilesUnix Include="$(MSBuildSrcDir)\**\proguard.sh"></MSBuildFilesUnix>
+    <MSBuildFilesUnix Include="$(MSBuildSrcDir)\libzip.5.0.$(LibExtension)"></MSBuildFilesUnix>
+  </ItemGroup>
+  <ItemGroup>
+    <MSBuildLibFiles Include="$(MSBuildSrcDir)\lib\**" Exclude="**\*.d.so;**\host-*\**"></MSBuildLibFiles>
+    <MSBuildLibFilesWinHost Include="$(MSBuildSrcDir)\lib\host-mxe-Win64\**" Exclude="**\*.d.dll"></MSBuildLibFilesWinHost>
+    <MSBuildLibFilesDarwinHost Include="$(MSBuildSrcDir)\lib\host-Darwin\**" Exclude="**\*.d.$(LibExtension)"></MSBuildLibFilesDarwinHost>
+  </ItemGroup>
+  <ItemGroup>
+    <TargetsFiles Include="$(RootBuildDir)\lib\xamarin.android\xbuild\**\MonoDroid.FSharp.targets"></TargetsFiles>
+    <TargetsFiles Include="$(RootBuildDir)\lib\xamarin.android\xbuild\**\Novell.MonoDroid.Common.targets"></TargetsFiles>
+    <TargetsFiles Include="$(RootBuildDir)\lib\xamarin.android\xbuild\**\Novell.MonoDroid.CSharp.targets"></TargetsFiles>
+    <TargetsFiles Include="$(RootBuildDir)\lib\xamarin.android\xbuild\**\Xamarin.Android.Bindings.targets"></TargetsFiles>
+  </ItemGroup>
+    <ItemGroup>
+    <VersionFiles Include="$(RootBuildDir)\Version*"></VersionFiles>
+  </ItemGroup>
+  <Target Name="GetXAVersion"
+      DependsOnTargets="GetXAVersionInfo"
+      Returns="$(XAVersion)">
+    <PropertyGroup>
+      <XAVersion>$(ProductVersion).$(XAVersionCommitCount)</XAVersion>
```

Comment:
This should be `$(ProductVersion)-$(XAVersionCommitCount)` (use `-`, not `.`), as that's what we currently do:

```
$ ls -l /Library/Frameworks/Xamarin.Android.framework/Versions/Current
lrwxr-xr-x  1 root  wheel  8 Feb  4 16:03 /Library/Frameworks/Xamarin.Android.framework/Versions/Current -> 9.1.0-38
```

Reply:
@jonpryor I copied this target over from https://github.com/xamarin/xamarin-android/blob/8e73664936f6cf57e3e5f7f22227ec82370cf55c/build-tools/create-vsix/create-vsix.targets#L125 in an effort to eventually remove that target from the create-vsix.targets file, and re-use this one. Is there a reason we can't change the .pkg version formatting now for consistency across macOS and Windows?

Reply:
We should be able to change the version formatting, yes.